### PR TITLE
update zh ingress.md

### LINKED ISSUE #18323: update zh ingress
**This is a Bug Report**

file: content/zh/docs/concepts/services-networking/ingress.md

**Problem:**
"ingress 是什么？和 跨可用区失败" 出现多次
**Proposed Solution:**
delete
**Page to Update:**
https://kubernetes.io/zh/docs/concepts/services-networking/ingress/

### PR DESCRIPTION
**Pull request by QizhouYang**
on master: fixes #18323

Address: https://kubernetes.io/zh/docs/concepts/services-networking/ingress/#

### CODE REVIEW

**Comment by QizhouYang** [comment]
I signed it

**Comment by QizhouYang** [comment]
I signed it

File: content/zh/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -63,8 +63,31 @@ Cluster network
 Service
 : A Kubernetes {{< glossary_tooltip term_id="service" >}} that identifies a set of Pods using {{< glossary_tooltip text="label" term_id="label" >}} selectors. Unless mentioned otherwise, Services are assumed to have virtual IPs only routable within the cluster network. 
 -->
+<<<<<<< HEAD
```
Comment (SataQiu) [inline content/zh/docs/concepts/services-networking/ingress.md]: 需要解决下冲突

**Comment by QizhouYang** [comment]
@SataQiu 您好，我本地代码和我的远程仓库都未有冲突代码啊！

**Comment by sftim** [comment]
@QizhouYang would you be willing to rebase this pull request?

I think https://kubernetes.io/zh/docs/contribute/intermediate/#%E4%BD%BF%E7%94%A8%E6%9C%AC%E5%9C%B0%E5%85%8B%E9%9A%86 explains the steps to follow.

**Comment by QizhouYang** [comment]
@markthink  Thanks, please review

**Comment by xichengliudui** [comment]
The commit message need to be update

**Comment by QizhouYang** [comment]
@xichengliudui Thanks，what to do and how to update commit message?

**Comment by QizhouYang** [comment]
@xichengliudui   please review the code，Thanks！

**Pull request merged**